### PR TITLE
homepage video

### PR DESCRIPTION
**Pull request by EilidhIsla**
Adds an option for the hero card to be a video, and adds a video option to the post-driven feature block

### CODE REVIEW

**Comment by EilidhIsla** [comment]
Sorted formatting, set max line length to 0

**Comment by EilidhIsla** [comment]
Also added a new field to homepage feature post category, `feature-video-text`

File: 4.0/css/nihr-new.css [hunk 1]
```diff
@@ -1141,6 +1145,10 @@ section.feed {
   display: none;
 }
 
+.card__full-width-image {
```
Comment (lsp01) [inline 4.0/css/nihr-new.css:1148]: @EilidhIsla I wonder if we just have a padding--none class at this point? I think there have a few times I might have looked for it as we have similar for margins.

Comment (EilidhIsla) [reply 4.0/css/nihr-new.css:1148]: @lsp01 Good shout, I'll update.

**Review by lsp01 — APPROVED** [review]
All looks good!

**Pull request merged**